### DISCUSSION TITLE
[Design] Trash Mechanics

### DISCUSSION THREAD

**Discussion by kugamo** [ParadiseSS13/Paradise#19219] [Documents]
![image](https://user-images.githubusercontent.com/16618648/193366733-94d64fb4-3fcf-45fd-bb7d-1581e432f183.png)

Made this a while ago, thinking about implementing it. Welcoming review from the design team on whether or not its viable as described here, or what would need to change in order to work on paradise.

Could be assumed to be part of the janitors duties, a new service role, or a new cargo role. Unsure.

**Comment by kugamo** [comment]
Working on the PR right now. No new roles or anything, just the new mechanics. Also I forgot to mention trashbags will be able to be broken if you open them up too many times or take out items too many times, this is to make function like trashbags and not trash scoops (no more picking up and trashing infinite amount of items) plus it will give the rotting mechanics more of a field to play in.

**Comment by Sirryan2002** [comment]
In support of this
